src/cmap/establish/test.rs

```diff
@@ -70,6 +71,7 @@ async fn speculative_auth_test(
         doc! { "find": "foo", "limit": 1  },
     );
 
+    // TODO: will have to add versioned API options before running command
```

Comment:
We generally put ticket numbers on all of our TODOs that we commit to the driver. Am I correct in thinking that this is blocked on us having a new enough server version in our evergreen matrix to support this feature? If so, I can file a ticket for adding that to the matrix and then you can put the ticket number here.

Reply:
This is currently blocked on not being able to declare a server API version for the test runner. Drivers will eventually have to run their tests against a server with `requireApiVersion` enabled, which will require being able to pass in an API version through environment variables (as it can't be declared using the connection string). However, that is not possible yet as mongo-orchestration doesn't support starting a server with `requireApiVersion` enabled (due to its internal `isMaster` check not declaring an API version, which leads it to believe that it wasn't able to start the cluster).

I can either file an issue pre-emptively (which may become obsolete if we were to create a DRIVERS ticket for this) and add that number here, or I can drop the TODO comments, as I only added them to highlight any commands that will not work on a server with `requireApiVersion` enabled.

Reply:
Assuming the tests will fail when run against a server with requireApiVersion before we make any code changes, a TODO comment probably isn't needed. If they will silently pass though, we definitely want a comment with a ticket number so that we can keep track of the fact that need to do extra work whenever it becomes unblocked.